### PR TITLE
Turbo version worklet start stop

### PR DESCRIPTION
**Pull request by karol-bisztyga**

### CODE REVIEW

File: Common/cpp/SharedWorkletStarter.cpp [hunk 1]
```diff
@@ -26,7 +35,93 @@ SharedWorkletStarter::~SharedWorkletStarter() {
 }
 
 jsi::Value SharedWorkletStarter::asParameter(jsi::Runtime &rt) {
-  return jsi::Value(this->id);
+  
+  class HO : public jsi::HostObject {
+    public:
+    int id;
+    SharedWorkletStarter *starter;
```
Comment (Szymon20000) [inline Common/cpp/SharedWorkletStarter.cpp]: Why not std::shared_ptr ?

File: Common/cpp/SharedWorkletStarter.cpp [hunk 2]
```diff
@@ -26,7 +35,93 @@ SharedWorkletStarter::~SharedWorkletStarter() {
...
+            const jsi::Value *args,
+            size_t count
+        ) -> jsi::Value {
+          if (starter->unregisterListener != nullptr) {
+            (*starter->unregisterListener)();
+            starter->unregisterListener = nullptr;
+          }
+          return jsi::Value::undefined();
+        };
+        return jsi::Function::createFromHostFunction(rt, name, 1, callback);
```
Comment (Szymon20000) [inline Common/cpp/SharedWorkletStarter.cpp]: Start and Stop don't have any args.

File: Common/cpp/WorkletModule.cpp [hunk 3]
```diff
@@ -129,4 +76,4 @@ void WorkletModule::setApplierId(int applierId) {
   this->applierId = applierId;
 }
 
-int WorkletModule::applierId = INT_MAX;
+int WorkletModule::Applier_Id = INT_MAX;
```
Comment (Szymon20000) [inline Common/cpp/WorkletModule.cpp]: Maybe we should move it to the ApplierRegistry? 
New_Local_Applier_Id?

File: Common/cpp/SharedWorkletStarter.cpp (hunk 1 above)
Comment (karol-bisztyga) [reply Common/cpp/SharedWorkletStarter.cpp]: because `this` is passed down to the inner class

File: Common/cpp/WorkletModule.cpp (hunk 3 above)
Comment (karol-bisztyga) [reply Common/cpp/WorkletModule.cpp]: fixed 957dc94

File: Common/cpp/SharedWorkletStarter.cpp (hunk 2 above)
Comment (karol-bisztyga) [reply Common/cpp/SharedWorkletStarter.cpp]: fixed 957dc94

**Pull request merged**